crates/api_common/src/utils.rs

```diff
@@ -1004,26 +1004,25 @@ pub(crate) async fn proxy_image_link(link: Url, context: &LemmyContext) -> Lemmy
 }
 
 pub async fn proxy_image_link_opt_api(
-  link: &Option<String>,
+  link: Option<Option<DbUrl>>,
```

Comment:
Is there anything preventing us from using a single layer of `Option` here?

Reply:
These ones I left as helper functions, because of the internal async call, and not being able to use async map. The one right below this uses the single option version.